.gitignore

```diff
@@ -17,3 +17,6 @@
 
 # Ignore uploaded files
 /public/uploads
+
+# Ignote simplecov coverage
+/coverage
```

Comment:
Wait, my bad. It's not an extension, it's a SublimeText core configuration option:

```json
"ensure_newline_at_eof_on_save": true
```